packages/main/src/themes/ColorPicker.css

```diff
@@ -62,9 +62,12 @@
     width: 0.9375rem;
     height: 1.5rem;
     background: transparent;
+    box-sizing: border-box;
+}
+
+[ui5-slider]::part(handle-container) {
```

Comment:
The handle appears vertically miss-aligned when focused. Turns out those styles are redundant in this case. I've tested adding the following selector and it fixes the display:

```
[ui5-slider]::part(handle-container):focus-within {
    margin-inline-start: unset;
    margin-top: unset;
}
```